README.md

```diff
@@ -11,26 +11,39 @@ None of these branches have a lib/cfg/gd_config.yml file, but you always need on
 ### Working on a new storm branch
 
 To set up the branch:
-1. Create a new RStudio project just for this storm. Clone the GitHub repository into this project. Because it's a separate folder, you'll be able to add storm-specific data and image files to this folder/project without overwriting files from other storms.
-1. Create and switch to a new storm-specific branch. Use the branch naming convention `Stormname-noaacode`, e.g., `Matthew-al142016`.
-1. Edit .gitconfig so that the following files are *not* ignored: `*.ind`, `build/status/*.yml`, `viz_config.yml`, `lib/cfg/gd_config.yml`, `2_process/src/filter_sites_custom.R`.
-1. Add a viz_config.yml. See other storm branches for examples.
-1. Add a 2_process/src/filter_sites_custom.R file and function. See other storm branches for examples.
+1. Create a new folder and RStudio project just for this storm. Clone the GitHub repository into this project. Because it's a separate folder, you'll be able to add storm-specific data and image files to this folder/project without overwriting files from other storms.
+1. Create and switch to a new storm-specific branch. Use the branch naming convention `Stormname-year`, e.g., `Matthew-2016`.
+1. Edit .gitignore so that the following files are *not* ignored: `*.ind`, `build/status/*.yml`, `viz_config.yml`, `lib/cfg/gd_config.yml`, `2_process/src/filter_sites_custom.R`.
+1. Add a viz_config.yml. See other storm branches and viz_config-example.yml for examples.
+1. Add a 2_process/src/filter_sites_custom.R file and function. See other storm branches and 2_process/src/filter_sites_custom-example.R for examples.
 1. Create a shared Google Drive folder for the cache in https://drive.google.com/drive/u/1/folders/169KSGMULk6eJxBTsIDQ0l9z65CueZEbF. Run `scipiper::gd_config(config_file=target_name, folder=I("drive-folder-id"))`.
```

Comment:
Are these two steps? Or is `scipiper::gd_config` what you run to create the new folder? I definitely just went to that drive and said "New Folder" then named it.

Reply:
Two steps - you run gd_config (or just edit viz/cfg/gd_config.yml) _after_ using the New Folder option of Drive. Will edit.